aiocache/factory.py

```diff
@@ -172,12 +172,12 @@ def get(self, alias: str) -> object:
         self._caches[alias] = cache
         return cache
 
-    def create(self, alias: str, **kwargs):
+    def create(self, alias, **kwargs):
         """Create a new cache.
 
         You can use kwargs to pass extra parameters to configure the cache.
 
-        :param alias: alias to pull configuration from
+        :param alias (str): alias to pull configuration from
```

Comment:
Not sure why this file has been changed, seems unrelated.

Reply:
It was a mypy error.  Should I revert it and submit a separate PR?

Reply:
Ignore the existing 12 errors:
https://github.com/aio-libs/aiocache/actions/runs/4235831591/jobs/7359931126

They can be dealt with separately. I think we just want to remove this entire file, as I don't see what benefit this part adds (but makes the code unnecessarily complex and fail type checking).

Reply:
OK, I'll remove the extraneous mypy fixes from this PR.